pkg/executor/api.go

```diff
@@ -61,7 +62,29 @@ func (executor *Executor) getServiceForFunctionApi(w http.ResponseWriter, r *htt
 		return
 	}
 
-	serviceName, err := executor.getServiceForFunction(fn)
+	t := fn.Spec.InvokeStrategy.ExecutionStrategy.ExecutorType
```

Comment:
I noticed that we are getting function from K8S API - which is in the request path - should we raise an issue for improving that performance?

Reply:
https://github.com/fission/fission/issues/1713